pom.xml

```diff
@@ -454,6 +455,16 @@
         <artifactId>log4j-over-slf4j</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.mongodb</groupId>
+        <artifactId>mongodb-driver-sync</artifactId>
```

Comment:
Why are we using the sync driver? It seems like several patterns may benefit from using an async. We definitely use async for something in the dynamo implementation.

Reply:
sync driver has been remove. We now use ReactiveStreams.